src/WebJobs.Script.WebHost/Standby/StandbyOptionsSetup.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    internal class StandbyOptionsSetup : IConfigureOptions<StandbyOptions>
+    {
+        private readonly IEnvironment _environment;
+
+        public StandbyOptionsSetup(IEnvironment environment)
+        {
+            _environment = environment ?? throw new ArgumentNullException(nameof(environment));
+        }
+
+        public string Name => throw new NotImplementedException();
```

Comment:
Why is Name here? It's not on the base interface from what I can see. Our other setup classes don't implement Name.

Reply:
Not sure... must have been from some refactoring. Will remove.